src/mapOfPeople.js

```diff
@@ -40,7 +40,20 @@
  * @return {Map}
  */
 function mapOfPeople(people) {
-  // write code here
+  const map = new Map();
+  let timeSec = 0;
+
+  for (const person of people) {
+    if (!map.has(person)) {
+      map.set(person, timeSec);
+      timeSec++;
+    } else {
+      map.delete(person);
+      timeSec++;
+    }
+  }
```

Comment:
```suggestion

  for (let seconds = 0; seconds < people.length; seconds++) {
    if (!map.has(people[seconds])) {
      map.set(person, seconds);
    } else {
      map.delete(people[seconds]);
    }
  }
```

You can use a much more simple variant of the solution and get rid of the unnecessary variable 'timeSec'.
Just FYI